spec/models/fixity_check_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+describe FixityCheck do
+
+  let(:good_asset)    { build(:asset_image_with_correct_sha512) }
+
+  let(:good_asset_2)  { build(:asset_mp3_with_correct_sha512) }
+  # :asset_with_faked_file, by factory default,comes with a random fake sha512.
+  let(:corrupt_asset) { build(:asset_with_faked_file) }
+
+  it "can check an asset's fixity sha512" do
```

Comment:
So, normally rspec recommends you split things into separate indpendent tests for readability and so when one fails it's clear _what_ failed. 

Sometimes we violate that when the test setup is so expensive and we don't want it to be run for each test. 

That _might_ be true here. 

But think it may not be worth it here regardless, this is an awful lot  in one `it` example, it makes it hard to figure out what it's checking, and if one of the tests fail in the future it may make it confusing what functionality broke. Split into more tests I think?